apps/cookbook/src/app/examples/default-list-example/default-list-example.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+const config = {
+  template: `<kirby-page
+></kirby-page>`,
+  styles: [
+    `li::before {
+      display: none;
+    }`,
```

Comment:
Indentation seems to be a little off when displayed:

![image](https://user-images.githubusercontent.com/150451/134640516-d8eaf730-b42b-48d6-ad1e-ae75b587a2bc.png)